common/genes/wc_gene_facial_markings.txt

```diff
@@ -845,6 +845,225 @@
+				female = male
+				boy = male
+				girl = male
+			}
+
+			nathrezim_male_02 = {
+				index = 33
+				male = {
+					decal = {		
+						body_part = head
+						textures = {
+							diffuse = "gfx/models/portraits/decals/makeup/decal_nathrezim_male_02.dds"
+						}
+						blend_modes = {				 #overlay/replace/hard_light/multiply
+							diffuse = replace
+							# normal = overlay
+							# properties =
+						}
+						alpha_curve = {
+							{ 0  0 }
+							{ 1  @max_alpha_blood }
+						}
+						decal_apply_order = post_skin_color
+						priority = 10
+					}
+				}
+				female = male
+				boy = male
+				girl = male
+			}
+
+			nathrezim_male_03 = {
+				index = 34
+				male = {
+					decal = {		
+						body_part = head
+						textures = {
+							diffuse = "gfx/models/portraits/decals/makeup/decal_nathrezim_male_03.dds"
+						}
+						blend_modes = {				 #overlay/replace/hard_light/multiply
+							diffuse = replace
+							# normal = overlay
+							# properties =
+						}
+						alpha_curve = {
+							{ 0  0 }
+							{ 1  @max_alpha_blood }
+						}
+						decal_apply_order = post_skin_color
+						priority = 10
+					}
+				}
+				female = male
+				boy = male
+				girl = male
+			}
+
+			nathrezim_female_02 = {
+				index = 35
+				male = {
+					decal = {		
+						body_part = head
+						textures = {
+							diffuse = "gfx/models/portraits/decals/makeup/decal_nathrezim_female_02.dds"
+						}
+						blend_modes = {				 #overlay/replace/hard_light/multiply
+							diffuse = replace
+							# normal = overlay
+							# properties =
+						}
+						alpha_curve = {
+							{ 0  0 }
+							{ 1  @max_alpha_blood }
+						}
+						decal_apply_order = post_skin_color
+						priority = 10
+					}
+				}
+				female = male
+				boy = male
+				girl = male
+			}
+
+			nathrezim_female_03 = {
+				index = 36
+				male = {
+					decal = {		
+						body_part = head
+						textures = {
+							diffuse = "gfx/models/portraits/decals/makeup/decal_nathrezim_female_03.dds"
+						}
+						blend_modes = {				 #overlay/replace/hard_light/multiply
+							diffuse = replace
+							# normal = overlay
+							# properties =
+						}
+						alpha_curve = {
+							{ 0  0 }
+							{ 1  @max_alpha_blood }
+						}
+						decal_apply_order = post_skin_color
+						priority = 10
+					}
+				}
+				female = male
+				boy = male
+				girl = male
+			}
+
+			nathrezim_male_04 = {
+				index = 37
+				male = {
+					decal = {		
+						body_part = head
+						textures = {
+							diffuse = "gfx/models/portraits/decals/makeup/decal_nathrezim_male_04.dds"
+						}
+						blend_modes = {				 #overlay/replace/hard_light/multiply
+							diffuse = replace
+							# normal = overlay
+							# properties =
+						}
+						alpha_curve = {
+							{ 0  0 }
+							{ 1  @max_alpha_blood }
+						}
+						decal_apply_order = post_skin_color
+						priority = 10
+					}
+				}
+				female = male
+				boy = male
+				girl = male
+			}
+		}
+
+		gene_facial_markings_2 = {
```

Comment:
The gene need localization too.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132095634-5b96c28d-f4f8-4e94-a2db-df3cc017b278.png)

</details>

Reply:
Added in 1fc763f0705c037c6b51d4784753c5e10e3d8d9e.

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/132096384-ff749198-b7b1-4d4d-b35d-57f187cb9b82.png)

</details>